src/nls/root/strings.js

```diff
@@ -574,6 +574,7 @@ define({
     "CMD_REFRESH_WINDOW"                        : "Reload With Extensions",
     "CMD_RELOAD_WITHOUT_USER_EXTS"              : "Reload Without Extensions",
     "CMD_NEW_BRACKETS_WINDOW"                   : "New {APP_NAME} Window",
+    "CMD_LAUNCH_SCRIPT_MAC"                     : "Install Launch Script",
```

Comment:
Could we change this to "Add Brackets Command Line"?

It's 20 characters versus 26 but I think it will still fit in the menu because there is extra space there.